webapp/src/components/Metrics/PerformanceAnalysisTable.tsx

```diff
@@ -0,0 +1,658 @@
+              } ) with:`}
+              labelPlacement="start"
+              sx={{ gap: 1, paddingRight: 2 }}
+              control={
+                <PipelineSelect
+                  selectedPipeline={comparedPipeline ?? ""}
+                  onChange={(value) => setComparedPipeline(value)}
+                  pipelineMenuItem={config.pipelines?.map(
+                    ({ name }, pipelineIndex) =>
+                      pipelineIndex !== pipeline.pipelineIndex && (
+                        <MenuItem key={pipelineIndex} value={pipelineIndex}>
+                          {name}
+                        </MenuItem>
+                      )
+                  )}
+                />
+              }
+            ></FormControlLabel>
+          </Box>
+        )}
+      </Box>
+      <Box
+        border={(theme) => `1px solid ${theme.palette.grey[200]}`}
+        sx={{ backgroundColor: (theme) => theme.palette.background.paper }}
+      >
+        <Table
+          sx={{
+            "& .MuiDataGrid-cell": {
+              borderBottom: "none",
+            },
+            "& .MuiDataGrid-columnHeaders": {
+              borderBottom: "none",
+            },
+            "& .MuiDataGrid-virtualScroller": {
+              // Stops accidental navigation on horizontal scroll with touch pad
+              overscrollBehaviorX: "contain",
+            },
+            "& .overall": {
+              borderRight: (theme) => `1px solid ${theme.palette.grey[300]}`,
+              background: (theme) => theme.palette.grey[200],
+            },
+            "& .delta": {
+              background: (theme) => theme.palette.grey[100],
+            },
+            "& .content-sticky": {
+              position: "sticky",
+              left: 0,
+              zIndex: (theme) => theme.zIndex.mobileStepper,
+              background: (theme) => theme.palette.background.paper,
+            },
+            "& .content-sticky-overall": {
+              position: "sticky",
+              left: 0,
+              background: (theme) => theme.palette.grey[200],
+            },
+            "& .no-border-right": {
+              borderRight: "none",
+            },
+            "& .border-left": {
+              borderLeft: (theme) => `1px solid ${theme.palette.grey[300]}`,
+            },
+          }}
+          showCellRightBorder
+          showColumnRightBorder
+          sortModel={sortModel}
+          onSortModelChange={setSortModel}
+          columnVisibilityModel={columnVisibilityModel}
+          onColumnVisibilityModelChange={setColumnVisibilityModel}
+          getRowClassName={({ id }) =>
+            `${id === OVERALL_ROW_ID ? "overall" : ""}`
+          }
+          getCellClassName={({ id, field }) =>
+            `${
+              id === OVERALL_ROW_ID
+                ? "content-sticky-overall"
+                : field.startsWith("delta")
+                ? "delta"
+                : ""
+            }`
+          }
```

Comment:
4 small things to simplify this bit:
- Move the `& .overall` definition down below the `& .content-sticky` definition, so it has priority
- Add `cellClassName: "delta"` to all delta columns (3 places)
- Remove `& .content-sticky-overall`
- Use only the `getCellClassName` to add the `overall` class, as follows:
```suggestion
          getCellClassName={({ id }) =>
            id === OVERALL_ROW_ID ? "overall" : undefined
          }
```
That's 11 lines shorter and easier to follow.